### PR TITLE
I replaced the old form with a temp

### PR DESCRIPTION
**Pull request by Akfoz-s**
Hey, I was confused on some things so I'm not sure I did them right, (PLEASE let me know if not). This is a trial run for me. Replaced old form with a google form linked to my email account so at least someone's getting the responses, will change to SBU Crew gmail account shortly.

### CODE REVIEW

**Comment by vpaladino778** [comment]
This looks perfect, exactly what we needed. Could you just attach a screenshot of what the website looks like on that page?

**Comment by Akfoz-s** [comment]
Took a few screenshots to capture it all, also I filled the form out already for funsies so there's my "response" visible
Website1
Website2
Website3
Website4
Website5

File: prospectiverowers.html [hunk 1]
```diff
@@ -59,60 +59,16 @@
         </p>
     </div>-->
 
-    <header class="main-image text-center text-white d-flex">
+    <header class="main-image text-center text-white d-flex"> <!-- for me (zofka) this is the main image lol-->
```
Comment (vpaladino778) [inline prospectiverowers.html]: Lets remove this comment here so it's not in the live website code

File: prospectiverowers.html [hunk 2]
```diff
@@ -59,60 +59,16 @@
...
-            <!--<small id="rower2K" class="form-text text-muted">We'll never share your email with anyone else.</small>-->
-        </div>
-        <div class="form-group">
-            <label for="rowerContactMeInput">Would you like to be added to the email contact list and recieve crew updates?</label>
-            <input type="text" name="Would You like to be added to our mailing?" class="form-control" id="rowerContactMeInput" aria-describedby="rowerContactMe" placeholder="Yes/No">
-            <!--<small id="rower2K" class="form-text text-muted">We'll never share your email with anyone else.</small>-->
-        </div>
-        <button type="submit" class="btn btn-primary">Submit</button>
-    </form>
+    <!-- <h2 class="form-title-local">Prospective Rower Interest Form</h2> -->
```
Comment (vpaladino778) [inline prospectiverowers.html]: If we no longer need this, we can delete it.

**Review by vpaladino778 — COMMENTED** [review]
Looks really good, nice work. Just a few small requests then we can merge this in

File: prospectiverowers.html (hunk 2 above)
Comment (Akfoz-s) [reply prospectiverowers.html]: Gotcha, I'll try to resolve both issues after class. To update it, do I have to go back to my code and then make a new pull request, or...?

Comment (vpaladino778) [reply prospectiverowers.html]: So after you make your changes, you can do the following commands

`git add .` to add all changed files
`git commit -m "Message here"` This bunches up your changes into a 'commit', which is basically just a group of changes
`git push` This pushes the commits up to the 'remote' git repository AKA GitHub Repo

After you do all that, you'll see the changes reflected in this pull request

Comment (Akfoz-s) [reply prospectiverowers.html]: ...I think I did it. Did I?

**Review by vpaladino778 — APPROVED** [review]
Looks good! Nice work! Feel free to hit the merge button when you're ready, and it should be shown on stonybrookcrew.com

**Comment by Akfoz-s** [comment]
Awesome! Next on my list will be to change the "recruiting for 2018" banner.

Also, when I type "stonybrookcrew.com" it tells me the domain is free. Is that something that I should be concerned about?

**Pull request merged**